src/veranda/io/geotiff.py

```diff
@@ -24,6 +24,8 @@
 from osgeo import gdal
 from osgeo import gdal_array
 
+# TODO: @Sebhahn: should we steer this?
```

Comment:
What is this?

Reply:
That TODO related to `gdal.SetCacheMax(1024**3)`. I have removed it.